tools/build.bat

```diff
@@ -0,0 +1,14 @@
+@echo OFF
+set DEFAULT_ARGS="-DDSLINK_BUILD_EXAMPLES=ON -DDSLINK_BUILD_BROKER=OFF -DCMAKE_BUILD_TYPE=Release"
+set LOC="%cd%"
+cd  %LOC%/..
+echo "%cd%"
+if exist ./build/CMakeCache.txt (
+  echo "exist"
+) else (
+   del build
+   mkdir build
+)
+
+cd build
+cmake %DEFAULT_ARGS% ..
```

Comment:
The same script should also do the msbuild <Solution name> in my opinion, so that the .exe files are available. 
The other bat will pick the .exe and create zip.

Reply:
comments incorporated. Added patches to and "msbuild".